src/map/clif.c

```diff
@@ -17740,7 +17740,7 @@ void clif_display_pinfo(struct map_session_data *sd, int cmdtype) {
 		//0:PCRoom
 		details_drop[0] = 0;
 		//1:Premium
-		details_drop[1] = battle_config.vip_drop_increase;
+		details_drop[1] = (battle_config.vip_drop_increase * battle_config.item_rate_common) / 100;
```

Comment:
This is not completely correct, the client wants 1 rate, but we have 10 different item rates on server, so I used the most common one.